spec/spec_helper_local.rb

```diff
@@ -44,6 +44,10 @@ def param(type, title, param)
   param_value(catalogue, type, title, param)
 end
 
+def postgresql_version
+  '9.6'
+end
+
```

Comment:
This function does not seem to be used in the rest of the changes of this PR.  Was it added by mistake?